README.md

```diff
@@ -1,20 +1,71 @@
-# Docker image for Simply-E Patron Web Interface
-
-## Example docker-compose.yml
-
-```
-version: '2'
-services:
-  patron-web:
-    container_name: patron-web
-    image: lyrasis/patron-web:latest
-    environment:
-      CONFIG_FILE: /config
-    volumes:
-      - ./config:/config
-    ports:
-      - 3000:3000
-```
+# Docker Image for SimplyE Patron Web Interface
+
+This repository provides a working demonstration of the Library Simplified/SimplyE web-based client interface as a Docker image/container. You can find the parent codebase at <https://github.com/NYPL-Simplified/circulation-patron-web>.
+
+If you will be developing code for the project, you will need to build a local version of the Docker image to review your changes. The following section provides instructions for a local build. If you are not doing any development or making any code changes, but you just want to see the project running locally, you can skip to the section *Running a Container from the Image* below and use the second command there.
+
+## Building the Docker Image ##
+
+When you have code changes you wish to review locally, you will need to build a local Docker image with your changes included. There are a few steps to get a working build:
+
+1. Push your code changes into a separate git repository which is accessible for cloning in your current environment. As an example, let's assume your GitHub username is `nedgavlin`, that you've forked the parent repository into your account, and that you've pushed your code changes to a working branch named `my-code-changes` (used in step 6).
+
+2. Clone this repository into a clean directory in preparation for the build, then change into the repository directory:
+    ```
+    git clone https://github.com/lyrasis/docker-patron-web.git ./patronweb
+    cd patronweb
+    ```
+
+3. As instructed in the README for the parent project, you must have already configured development access to the Circulation Manager from this local application by adding a sitewide setting to the Circulation Manager. Review that document as needed.
+
+4. Review the file `config/cm-libraries.conf.sample` for proper formatting in creating a configuration file which links the web application to a library's SimplyE Circulation Manager in order to pull catalog feeds into the display. The SimplyE web client application can be configured on behalf of multiple libraries to provide a web-based client for their patrons. In a multi-tenant environment, each library will be represented in the config file as a separate line.
+
+5. You can create an empty config file or rename the sample file with the filename `config/cm-libraries.conf`. Here's an example entry in the file for a library with a Circulation Manager short name of `abclib` whose Circulation Manager is available at the domain `circulation-simplye.library.org`:
```

Comment:
Is this supposed to be `config/cm-libraries.conf.sample`?

Reply:
My thoughts aren't completely fleshed out on handling the config. In my other deployment repos, I've created samples for most other configs and not included the target file; there are either instructions for copying the sample to the primary file and editing (variables) or an Ansible template task that creates the appropriate file on the target from the variables files. For this particular file, the best way ultimately is to include the actual file with a commented example at the top. But I need to add a line to Amy's script that reads the file so it skips lines beginning with a #. I can make the change here to align with that goal. Does that sound good to you?